eventsdiscovery/src/main/java/com/schedjoules/client/eventsdiscovery/EventsDiscovery.java

```diff
@@ -84,4 +86,9 @@
      * @return
      */
     EventsDiscovery withStartAtOrAfter(DateTime startAtOrAfter);
+
+    /**
+     * Sets the event categories filter
+     */
+    EventsDiscovery withCategories(Iterable<Uri> categories);
```

Comment:
Why I did this way relates to the fact that the `label()` property of `Category` is a localized value, so it's a bit special. Normally we wouldn't put that into a model (or have a `Context`) parameter, but this comes from the backend, so it's a bit different. I am not even sure if this `Category` will fit well with some edge cases when user changes language. For example if we persisted categories with a different language. So I look at the `Category` model as a first approximation actually, to serve our current needs but it may change.
Other thing is that we simply only need the uri here for the request.
So these are the reasons why I didn't want to depend on `Category` here.

Request could be fired without knowing the localized labels. For example if SDK is opened right away with a category filter, we don't have labels at that point, that will be loaded parallel.